app/controllers/incomeSources/manage/ManageObligationsController.scala

```diff
@@ -169,7 +170,9 @@ class ManageObligationsController @Inject()(val checkSessionTimeout: SessionTime
         case Some(years) =>
           if (changeTo == "annual" || changeTo == "quarterly") {
             getIncomeSourceId(mode, incomeSourceId, isAgent = isAgent) match {
-              case Left(error) => showError(isAgent, {
+              case Left(error) =>
+                Logger("application").info(s"BEEP BEEP 1 ${error.getMessage}, mode: $mode, id: $incomeSourceId")
```

Comment:
this log message is not valuable for investigation, if you can provide controller details pls